### ISSUE TITLE
vue/html-indent. There is warn/error when container-tag contains only comments AND opening tag has more than one aligned attributes.

### ISSUE THREAD

**Issue by ZoolooS** [vuejs/eslint-plugin-vue#1743]
**Checklist**

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have read the [FAQ](https://eslint.vuejs.org/user-guide/#faq) and my problem is not listed.

**Tell us about your environment**

- **ESLint version:** 7.32.0
- **eslint-plugin-vue version:** 8.1.1
- **Node version:** v14.18.1
- **Operating System:** Win 10

**Please show your full configuration:**

```json5
{
  // this is not full configuration
  "parser": "vue-eslint-parser",
  "rules": {
    "vue/attribute-hyphenation": ["off", { "ignore": [] }],
    "vue/first-attribute-linebreak": "off",
    "vue/html-closing-bracket-newline": "off",
    "vue/html-indent": ["warn", 4, {
        "baseIndent": 0,
        "attribute": 1,
//        "closeBracket": 0,
        "alignAttributesVertically": true,
        "ignores": [
            "VElement[name=template].children",
            "VElement[name=thead].children",
            "VElement[name=tbody].children",
            "ConditionalExpression"
        ]
    }],
    "vue/html-comment-indent": ["error", 4],
    "vue/max-attributes-per-line": "off",
    "vue/multi-word-component-names": "off",
    "vue/multiline-html-element-content-newline": "off",
    "vue/singleline-html-element-content-newline": "off",
    "vue/no-unused-vars": "warn",
    "vue/require-prop-types": "warn",
    "vue/import/first": "off"
  }
}
```

**What did you do?**	

```vue
<template>
<section>

                <div>
                    <select id="code-viewer-main">
                        <!--<option value="">Boo01</option>-->  <!-- No warn -->
                        <option value="">Boo02</option>
                        <option value="">Boo03</option>
                    </select>

                    <button class="btn-copy"
                            @click="copy">
                        <!--Копировать-->    <!-- Warn -->
                    </button>
                </div>


                <div>
                    <select id="code-viewer-main"
                            @click="copy">
                        <!--<option value="">Boo01</option>-->    <!-- Warn -->
                        <!--<option value="">Boo02</option>-->
                        <!--<option value="">Boo03</option>-->
                    </select>

                    <button class="btn-copy">
                        <!--Копировать-->  <!-- No warn -->
                    </button>
                </div>
</section>
</template>
```

**What did you expect to happen?**
No warn/errors.

**What actually happened?**

There is warn/error when container-tag contains only comments AND opening tag has more than one aligned attributes.
![image](https://user-images.githubusercontent.com/11315202/145782474-40784edf-924f-4ff9-b14e-b86dd049e678.png)

**Repository to reproduce this issue**
